orc8r/cloud/go/obsidian/handler.go

```diff
@@ -227,7 +229,16 @@ func GetNetworkId(c echo.Context) (string, *echo.HTTPError) {
 	if nid == "" {
 		return nid, NetworkIdHttpErr()
 	}
-	return nid, CheckNetworkAccess(c, nid)
+	sanNid := sanitizeString(nid)
+	return sanNid, CheckNetworkAccess(c, sanNid)
```

Comment:
you can just do 
```
return sanNid, CheckNetworkAccess(c, sanitizeString(nid))
```
No need to create a temp variable since the name of the function is self explanatory